code/modules/reagents/Chemistry-Goon-Pyrotechnics.dm

```diff
@@ -25,9 +25,15 @@
 	result_amount = 4
 	required_temp = 424
 
+datum/reagent/clf3/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	M.adjustFireLoss(1*REM)
```

Comment:
sounds a bit too weak for *fucking ClF3*, at least make it do a bit more burn damage